### PR TITLE
feat: Made the github webhook content_type and insecure_ssl option types vars

### PR DESCRIPTION
**Pull request by sf-walsh**
## Description
Made the github webhook content_type and insecure_ssl option types vars

## Motivation and Context

Some environments may need to temporarily set environments up with a self-signed certificate and disabling SSL checks and the content type is a requirement for the github webhook to deliver successfully.

```
    content_type = var.webhook_content_type
    insecure_ssl = var.webhook_insecure_ssl
```

## Breaking Changes

No

## How Has This Been Tested?
- [X] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [X] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [X] I have executed `pre-commit run -a` on my pull request

Pre-commit check validations run prior to PR:
```
Terraform fmt............................................................Passed
Terraform validate.......................................................Passed
Terraform docs...........................................................Passed
Terraform validate with tflint...........................................Passed
check for merge conflicts................................................Passed
```

### CODE REVIEW

File: modules/github-repository-webhook/variables.tf [hunk 1]
```diff
@@ -38,3 +38,15 @@ variable "webhook_secret" {
...
+variable "webhook_insecure_ssl" {
+  description = "Disable verify SSL certificates when delivering payloads"
+  type        = bool
+  default     = false
+}
+
+variable "webhook_content_type" {
+  description = "Webhook Content Type"
+  type        = string
+  default     = "application/json"
```
Comment (dynamike) [inline modules/github-repository-webhook/variables.tf:51]: Looking at the [GitHub terraform provider docs](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository_webhook#content_type), valid values here would be `json` or `form`, which can be validated. 
```suggestion
  default     = "json"
  validation {                                                                                                                                                                        
    condition     = contains(["json", "form"], var.webhook_content_type)                                                                                                              
    error_message = "Valid values are either `json` or `form`"                                                                                                                        
  }
```

**Review by dynamike — CHANGES_REQUESTED** [review]
Generally looks good! I left one suggestion around validation inputs for the content type to match the GitHub terraform provider.

**Comment by sf-walsh** [comment]
@dynamike thanks for the info. Checked that over, added the changes and pushed them, thanks for that.

**Review by dynamike — APPROVED** [review]
@bryantbiggs This PR looks good to me, if you want to look it over and merge.

File: modules/github-repository-webhook/variables.tf [hunk 2]
```diff
@@ -38,3 +38,19 @@ variable "webhook_secret" {
...
+variable "webhook_insecure_ssl" {
+  description = "Disable verify SSL certificates when delivering payloads"
+  type        = bool
+  default     = false
+}
+
+variable "webhook_content_type" {
+  description = "Webhook Content Type"
+  type        = string
+  default     = "json"
```
Comment (nitrocode) [inline modules/github-repository-webhook/variables.tf:51]: Shouldn't this be left as the original default?

```suggestion
  default     = "application/json"
```

Comment (sf-walsh) [reply modules/github-repository-webhook/variables.tf:51]: @nitrocode Previously @dynamike referred me here [Github Webhook accepted vlaues](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository_webhook#content_type)

I have tested both ways, with "application/json" and with "json" and both work without issue. The reason I decided to keep with "json" over the previous default was due to the documentation stating the proper accepted inputs even though both work and are acceptable.

Comment (nitrocode) [reply modules/github-repository-webhook/variables.tf:51]: ah ok I see https://github.com/terraform-aws-modules/terraform-aws-atlantis/pull/312#discussion_r996105398

Thank you.

**Pull request closed without merging**